### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by spaceninja** [MLTSHP/mltshp-patterns#225]
Now that we have Storybook, at the very least we should review the a11y panel on each story to see if there are any a11y fixes to be made (I know there are some contrast violations for buttons, for example).

This issue is for reviewing all the a11y panels and filing new GitHub issues for each violation. Those issues should get the a11y label to make them easy to filter.
